data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json

```
@@ -75,5 +75,62 @@
     "min_range": 4,
     "max_range": 8,
     "range_increment": 0.12
+  },
+  {
+	"id": "cleansingflame_hermes_heal",
+	"type": "SPELL",
+	"spell_class": "CLEANSING_FLAME",
+	"name": "Hermes reborn",
```

```suggestion
	"name": "Hermes Reborn",
```
Hermes? I'm not familiar with a phoenix or similar being associated with Hermes. (Thoth is associated with the stork, IIRC.)

---

You get healed, after that got some aoe damage, and split damage after all - damage part make some debuff after using and dont give 100% healing